plan/plan.go

```diff
@@ -234,10 +234,11 @@ type Account struct {
 type Component struct {
 	ComponentCommon `yaml:",inline"`
 
-	Accounts  map[string]Account `yaml:"accounts"` // Reference accounts for remote state
-	Component string             `yaml:"component"`
-	EKS       *v2.EKSConfig      `yaml:"eks,omitempty"`
-	Env       string
+	Accounts        map[string]Account `yaml:"accounts"` // Reference accounts for remote state
+	AccountBackends map[string]Backend `yaml:"account_backends"`
```

Comment:
Maybe it's just me, but this name is a little confusing -- it's RemoteStateBackends rather than AccountBackends?

Reply:
Yeah it is, but needs a bit of a refactor to fix. I am trying to avoid that refactor for now.